libs/SalesforceSDKCore/SalesforceSDKCoreTests/CSFSalesforceOAuthRefreshTests.m

```diff
@@ -82,7 +84,11 @@ - (void)testRevokedToken {
     user.credentials.refreshToken = @"RefreshToken";
     user.credentials.instanceUrl = [NSURL URLWithString:@"http://example.org"];
     user.credentials.identityUrl = [NSURL URLWithString:@"https://example.org/id/orgID/userID"];
+    NSError *error = nil;
+    [[SFUserAccountManager sharedInstance] saveAccountForUser:user error:&error];
+     XCTAssertNil(error);
 
+    [SFUserAccountManager sharedInstance].currentUser = user;
```

Comment:
Is this necessary for the test to pass?  Because at first glance it doesn't seem that it should be, and if it is, I wonder why.